src/components/Context.ts

```diff
@@ -1,6 +1,11 @@
 import React from 'react'
+import type Subscription from '../utils/Subscription'
 
-export const ReactReduxContext = /*#__PURE__*/ React.createContext(null)
+export type ReduxContextProps = {
+  store: any
```

Comment:
:speech_balloon: Tell you what, let's use my favorite "stub this out" trick for converting stuff to TS.

~~Let's add a `type $FixTypeLater = any` somewhere, and use that here. That way we know this is something that needs to be addressed down the road.~~  

Or, actually... the TS typedefs for React-Redux should already have the right-ish types that we need, and they specifically reference the core Redux `Store` type here:

https://www.unpkg.com/browse/@types/react-redux@7.1.16/index.d.ts

```ts
export interface ReactReduxContextValue<SS = any, A extends Action = AnyAction> {
    store: Store<SS, A>;
    storeState: SS;
}
```


Hmm. Having said that, those types actually seem buggy, because we _don't_ put the `storeState` in there, and haven't seen v6.

Still, we ought to try to reuse those typedefs as much as possible, because a lot of this stuff has been figured out for us already :)  still a good idea to stub things out with `$FixTypeLater` in places, but we should be starting with these types as our baseline.

Reply:
I've added a new type `FixTypeLater` which is just an alias for `any` -- I'm not sure if I see all that much value in it (since you can just search for usages for `any` as well) but no strong feelings so don't mind adding it.

Good call on the @types -- I didn't think of using those. I've put them in and replaced the `storeState` field with `subscription`. Is this more like what you had in mind?

Reply:
Yep!